### PR TITLE
Added Wallet::blockchainHeight() and Wallet::daemonBlockchainHeight() to Wallet api

### PR DESCRIPTION
**Pull request by mbg033**

### CODE REVIEW

File: src/wallet/api/wallet.cpp [hunk 1]
```diff
@@ -413,6 +417,25 @@ uint64_t WalletImpl::unlockedBalance() const
...
+    if (!err.empty()) {
+        LOG_ERROR(__FUNCTION__ << ": " << err);
+        m_errorString = err;
+        m_status = Status_Error;
+    } else {
+        m_status = Status_Ok;
+        m_errorString = "";
+    }
+    return result;
+}
```
Comment (moneromooo-monero) [inline src/wallet/api/wallet.cpp]: That should probably return 0, to mark error

File: src/wallet/api/wallet.cpp [hunk 2]
```diff
@@ -428,14 +451,14 @@ void WalletImpl::refreshAsync()
     m_refreshCV.notify_one();
 }
 
-void WalletImpl::setAutoRefreshInterval(int seconds)
+void WalletImpl::setAutoRefreshInterval(int millis)
 {
```
Comment (moneromooo-monero) [inline src/wallet/api/wallet.cpp]: Max range is 24 days. Maybe large enough, but mabe needs sanity checking.

File: src/wallet/wallet2.cpp [hunk 3]
```diff
@@ -4053,6 +4053,34 @@ std::string wallet2::get_daemon_address() const
...
+    else if (res.status != CORE_RPC_STATUS_OK)
+    {
+      err = res.status;
+    }
+  }
+  else
+  {
+    err = "possibly lost connection to daemon";
+  }
+  return res.height;
```
Comment (moneromooo-monero) [inline src/wallet/wallet2.cpp]: doens't set err to "" or whatever on success

File: src/wallet/api/wallet.cpp (hunk 1 above)
Comment (mbg033) [inline src/wallet/api/wallet.cpp]: fixed

File: src/wallet/api/wallet.cpp (hunk 2 above)
Comment (mbg033) [inline src/wallet/api/wallet.cpp]: added check, limited upper bound as 1 minute

File: src/wallet/wallet2.cpp (hunk 3 above)
Comment (mbg033) [inline src/wallet/wallet2.cpp]: fixed

**Pull request merged**